source/teams.yaml

```diff
@@ -42,6 +42,20 @@
           schema:
             $ref: '#/definitions/TeamMap'
 
+  /teams/my_members:
```

Comment:
I think the channel equivalent is `/teams/{team_id}/channels/members`, maybe we copy that and do `/teams/members` for this?